geometry/SettingsViewController.swift

```diff
@@ -8,20 +8,18 @@
 
 import UIKit
 
-protocol SettingsViewControllerDelegate: class {
-    func settingsViewControllerDelegate(sender: SettingsViewController)
+protocol SettingsViewControllerDelegate {
+    func fillSettingsViewControllerDelegate(rColor: Float, gColor: Float, bColor: Float)
 }
 
 class SettingsViewController: UIViewController
 {
-    var settings = Settings()
+    //var settings = Settings()
     
-    weak var delegate: SettingsViewControllerDelegate?
-    
-    @IBAction func close(sender: AnyObject) {
-        dismissViewControllerAnimated(true, completion: nil)
-        delegate?.settingsViewControllerDelegate(self)
-    }
+    var delegate: SettingsViewControllerDelegate?
+    var redsvc = Float()
```

Comment:
What the 'svc' prefix is supposed to tell? This field is already inside of SettingsViewController class and it's not exposed to the outer world.